modules/objc/examples/code_snippets/GettingStarted.m

```diff
@@ -43,27 +55,29 @@ - (void)viewDidLoad {
     // Create a query to fetch documents of type SDK.
     CBLQueryExpression *type = [[CBLQueryExpression property:@"type"] equalTo:[CBLQueryExpression string:@"SDK"]];
     CBLQuery *query = [CBLQueryBuilder select:@[[CBLQuerySelectResult all]]
-                                          from:[CBLQueryDataSource database:database]
+                                          from:[CBLQueryDataSource collection: collection]
```

Comment:
I think we should stop showing QueryBuilder in the examples as it's not the maintenance mode now.

Reply:
this PR is with 3.1. I would keep this here as is, thinking we do not plan to release another dozen of 3.1s. I will make myself a note to strip this from 3.2.